app/main.py

```diff
@@ -1 +1,29 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_parts = command.split()
```

Comment:
Consider adding a check for the length of `command_parts` before unpacking it. This will prevent a `ValueError` if the command does not have exactly three parts.